src/lib/ngsiNotify/senderThread.h

```diff
@@ -61,6 +61,7 @@ typedef struct SenderThreadParams
   std::string                        fiwareCorrelator;
   struct timespec                    timeStamp;
   std::map<std::string, std::string> extraHeaders;
+  std::string                        subscriptionId;
```

Comment:
Not sure if this is related with all the "subscription id is already included in X" comments... Is this actually needed?